src/net/telemetry.rs

```diff
@@ -38,6 +38,8 @@ pub struct TelemetryBuffer {
     pub dacs: [DacCode; 2],
     /// The latest digital input states during processing.
     pub digital_inputs: [bool; 2],
+    /// The latest measured status related to pounder. See [PounderTelemetry].
+    pub pounder: Option<PounderTelemetry>,
```

Comment:
This doesn't need to exist in `TelemetryBuffer`, as there's no need to maintain the pounder telemetry at all points in the app like there is a need for the dacs and IOs. Instead, just have it exist in the final `Telemetry` structure.